app/views/api/v3/analytics/user_analytics/show.html.erb

```diff
@@ -14,178 +14,184 @@
+                         statistics: @statistics,
+                         data_table_name: registrations_table_name } %>
+  </div>
+
+  <!-- monthly follow-up patients view -->
+  <div class="card">
+    <h3>Follow-up<%= @statistics.dig(:metadata, :is_diabetes_enabled) ? ' ' : ' hypertension ' %>patients</h3>
+
+    <% follow_ups_table_name = 'follow_ups' %>
+
+    <%= render partial: 'gender_filter',
+               locals: { statistics: @statistics, data_table_name: follow_ups_table_name } %>
+
+    <%= render partial: 'monthly_gender_table',
+               locals: { resource: :follow_ups,
+                         user_analytics: @user_analytics,
+                         statistics: @statistics,
+                         data_table_name: follow_ups_table_name } %>
+  </div>
+
+  <!-- monthly hypertension control view -->
+  <div class="card">
+    <h3>Hypertension control</h3>
+    <p class="desc">
+      How many patients with hypertension had BP below 140/90 at their last follow-up visit in the month?
+    </p>
+
+    <table class="progress-table">
+      <% @user_analytics.monthly_period_list.each do |month_date| %>
+        <tr>
+          <td class="row-label no-padding"><%= month_date.strftime("%B") %></td>
+          <td><span class="percent">
                   <%=
                     @user_analytics.display_percentage(@statistics.dig(:monthly,
                                                                        :grouped_by_date,
+                                                                       :hypertension,
                                                                        :controlled_visits,
                                                                        month_date),
                                                        @statistics.dig(:monthly,
                                                                        :grouped_by_date,
+                                                                       :hypertension,
                                                                        :follow_ups,
                                                                        month_date))
                   %>
                 </span></td>
-            <td>
-              <%= zero_if_unavailable(@statistics.dig(:monthly,
-                                                      :grouped_by_date,
-                                                      :controlled_visits,
-                                                      month_date)) %>
-              of
-              <%= zero_if_unavailable(@statistics.dig(:monthly,
-                                                      :grouped_by_date,
-                                                      :follow_ups,
-                                                      month_date)) %>
-            </td>
-          </tr>
-        <% end %>
-      </table>
-    </div>
+          <td>
+            <%= zero_if_unavailable(@statistics.dig(:monthly,
```

Comment:
Same comments as above, given we are five layers deep in a nested structure, these should be moved to the presenter.